src/KW_checkbox.h

```diff
@@ -52,8 +49,18 @@ extern DECLSPEC KW_Widget * KW_CreateCheckbox(KW_GUI * gui, KW_Widget * parent,
  */
 extern DECLSPEC KW_bool KW_IsCheckboxChecked(KW_Widget * widget);
 
+/*
+ * \brief Sets the checked icon to be used
+ * \param widget the widget to be used
+ * \param Rect the icon sprite
+ */
 extern DECLSPEC void KW_CheckboxSetChecked(KW_Widget * widget, KW_Rect * Rect);
```

Comment:
I believe KW_CheckboxSetCheckedIcon and KW_CheckboxSetUncheckedIcon are better names.